docs/contributing/pull-requests/branching.md

```diff
@@ -51,22 +52,18 @@ Product.
 
 Choose a **Long-Lived Feature Branch** if the feature will:
 
-- Have multiple authors
-- Consist of multiple Pull Requests, which have already been reviewed
-- Require these multiple Pull Requests from multiple authors in order to produce the testable,
-  releasable change
+- Require multiple Pull Requests to produce a testable, releasable change, and
+- It is not possible to put the changes behind a feature flag
```

Comment:
🎨 I would enjoy even more language about how long-lived branches are to be avoided and worth a discussion if someone thinks one is needed.

Reply:
Open to suggestions or further refinement. 

The way I'm thinking about it is that this sums up the only reason to _have_ a long lived branch is because you need to keep it out of the main branch to be safe. The tip below makes it clear that you should avoid them.

Reply:
Fair enough, and this may be moot in practice.